### ISSUE TITLE
ios Support ? dit it done ?!

### ISSUE THREAD

**Issue by mahmoudelgharabawi** [rexthecoder/flutter_animated_splash_screen#11]

**Comment by IAMIbrahimmemon** [comment]
@rexthecoder I don't think it supports ios

**Comment by rexthecoder** [comment]
@IAMIbrahimmemon Yeah, it does not support IOS

**Comment by IAMIbrahimmemon** [comment]
> @IAMIbrahimmemon Yeah, it does not support IOS

on pub.dev it says it supports it

**Comment by rexthecoder** [comment]
> > @IAMIbrahimmemon Yeah, it does not support IOS
> 
> on pub.dev it says it supports it

Yeah, because I plan to add IOS support but when Flutter updated to the new splash, I noticed the load time for this was bad. Maybe in the next few days, I will work on a new flow which I think is better and has a better loading time(That would include IOS support)
